sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1078,6 +1090,8 @@ alterColumnAction
     | commentSpec
     | colPosition
     | setOrDrop=(SET | DROP) NOT NULL
+    | SET defaultExpression
```

Comment:
Do other databases support `SET DEFAULT expression`?

Reply:
Yes, it is part of the ANSI SQL.
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/1097932/156511258-88c5619a-38c6-46db-8ca6-4c510d052b11.png">